### ISSUE TITLE
[Screen Reader-Social Embedded-Create Profile]: TalkBack is not announcing the labels with edit box.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#9]
Priority: High

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Navigate to “First Name” edit box.
10. Verify the speech of talkback.

Expected
TalkBack should announce the labels with edit box. It should not announce “Edit” box editing.

Actual
TalkBack is not announcing the labels with edit box. It is announcing “Edit” box editing.

MAS Rule: MAS4.2.5

Impact
User who is rely on screen reader will not get complete information and might get confuse if labels are not associated or no instructions is defined for edit box.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back
